README.md

```diff
@@ -43,6 +44,13 @@ func main() {
 }
 ```
 
+2. This file may be used to test any recent local changes because
```

Comment:
Prefer to recommend the use of a [Go module reference `replace`](https://go.dev/doc/modules/managing-dependencies#local_directory) in order to test local changes, with the use of `go mod` dependencies management command, as follows...

```sh
go mod edit -replace github.com/heimweh/go-pagerduty=<path-to-local-version-of>/go-pagerduty
```

Reply:
As stated here: https://github.com/heimweh/go-pagerduty/pull/128#r1163063557

I think it's better to just leave out these suggestions. Folks should know how to run a go file and use a library already and everyone has a different way of doing it. Even the standard go way of doing things or even the links will change inevitably and then this will become out of date.